docusaurus/docs/Flutter/03-core-concepts/05-call-types.mdx

```diff
@@ -6,6 +6,27 @@ description: How Call Types control features and permissions
 
 import WithExternalLinks from '../../../shared/video/_withExternalLinks';
 
+When you create a call like this
+
+```dart
+final call = client.makeCall(callType: StreamCallType.Default(), "123")
+```
+
+You create a call of the type `default` with id `123`.
+
+There are 4 built-in call types, `default`, `audio_room`, `livestream` and `development`.
+
+You can also create your own types by calling `StreamCallType.custom`.
+  
+```dart
+final call = client.makeCall(callType: StreamCallType.custom("custom"), "123")
+```
+
+:::note
+  When using a custom call type, you need to make sure that the server is configured to handle the custom call type.
```

Comment:
maybe we should add a link or some kind of pointer to the Stream dashboard where user can do this?
so something like: https://dashboard.getstream.io/app/{you_app_id}/video/call-types/new

Reply:
I want to find a link to documentation for this ... will ask around.

Reply:
I embedded [this page](https://github.com/GetStream/stream-chat-docusaurus-cli/blob/production/shared/video/_call-types.m) according to the other SDKs